.github/workflows/data-pipeline.yml

```
@@ -0,0 +1,45 @@
+      POSTGRES_HOST: localhost
+      POSTGRES_PORT: '5432'
+      POSTGRES_DB: opendiscourse
+      POSTGRES_USER: opendiscourse
+      POSTGRES_PASSWORD: opendiscourse
+      GOVINFO_API_KEY: ${{ secrets.GOVINFO_API_KEY }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)